circle.yml

```diff
@@ -23,5 +23,4 @@ test:
     - bash .misc/deploy.coverage.sh:
         parallel: true
     - make -C docs clean
-    - sphinx-apidoc -f -o docs --no-toc coalib
-    - make -C docs html SPHINXOPTS=-W
+    - python setup.py docs
```

Comment:
Yes, `setup.py` doesnt currently set `SPHINXOPTS=-W`. The first option is for `setup.py` to always set `SPHINXOPTS=-W`.  If that causes problems, we can find better solution.